src/Components/Presentational/Splash.js

```diff
@@ -27,12 +30,15 @@ export default function Splash() {
   const theme = useTheme();
 
   useEffect(() => {
-    document.body.className = theme.palette.type === 'light' ? 'splash' : 'splash-dark';
-    return () => { document.body.className = ''; }
+    document.body.className =
+      theme.palette.type === 'light' ? 'splash' : 'splash-dark';
+    return () => {
+      document.body.className = '';
+    };
   });
 
   return (
-    <>
+    <Box className={classes.container}>
```

Comment:
Adding the box component seems to be unrelated from the spike. Don't know if this is being merged or not but just in case, was there any reason this change needed to be made here?

Reply:
Yeah this is a weird change! If you check out the `App.scss` file, I removed the global `text-align: center` line because it was overriding some of the react-diagrams styles and making the nodes render inconsistently. However, the Splash page still needed the center text, so I moved the style to live exclusively here.

Reply:
ahh gotcha makes sense! thanks!